<launch>

<node pkg="unity_ros_tf" type="odomfromtf.py" name="dog_odom_from_TF">
    <remap from="/odom_to_map" to="/global/odom_to_map"/>
    <param name="parent_frame" value="multijackal_02" />
    <param name="child_frame" value="multijackal_02/odom" />
</node>

<!-- <node pkg="unity_ros_tf" type="odomfromtf.py" name="dog_map_from_TF">
    <remap from="/odom_to_map" to="/dogmap_to_world"/>
    <param name="parent_frame" value="multijackal_01/map" />
    <param name="child_frame" value="multijackal_01/map" />
</node> -->

</launch>